src/services/User.ts

```diff
@@ -37,4 +37,79 @@ async function postUserDetails(userForm: UserProfile): Promise<
   }
 }
 
-export default postUserDetails;
+async function postUserFullName(full_name: string): Promise<
```

Comment:
Should have a more specific name. Right now the name says: "we're posting the user full name". Ok, but what to do with it?

Suggestion: `updateUserName` or `updateUserFullName`